doc/src/build/custom-upgrade-policy.md

```diff
@@ -979,26 +1002,15 @@ Array [
 ]
 ```
 
-Now, the emitted Event's `x` field has value `42` (changed from the
-original `41`).
+Now, the `Events` section emitted for the `x` field has a value of `42` (changed from the original `41`).
 
-If you wait a further day, change the constant again, and re-run the
-script, you will receive an output similar to the following, which
-indicates that the upgrade aborted with code `2` (`EDayNotAllowed`):
+If you attempt the first upgrade before Tuesday or you change the constant again and try the upgrade the following day, the script receives a response that includes an error similar to the following, which indicates that the upgrade aborted with code `2` (`EDayNotAllowed`):
```

Comment:
This was my mistake from before, but I got the name of the error constant wrong here.

```suggestion
If you attempt the first upgrade before Tuesday or you change the constant again and try the upgrade the following day, the script receives a response that includes an error similar to the following, which indicates that the upgrade aborted with code `2` (`ENotAllowedDay`):
```